src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/constants/TableMarkupConstants.java

```diff
@@ -7,8 +7,10 @@ public final class TableMarkupConstants {
     public static final String RESPONSE_TABLE_HEADER = "|Response %s | |\r\n";
     public static final String SUPPORTING_MATERIAL_TABLE_HEADER = "|Supporting material | %s|\r\n";
     public static final String ORDER_APP_HEARING_MARKUP = "|Hearing | %s|\r\n";
-    public static final String ORDER_APP_DOC_MARKUP = "|Description | %s|\r\n"
+    public static final String ORDER_APP_DOC_MARKUP = "|Description  %s|\r\n"
```

Comment:
minor: could we choose a better name for this? its not very clear from name alone